apis/userapi/userapi.yaml

```diff
@@ -1721,6 +1721,98 @@ paths:
             'INTERNAL SERVER ERROR. We track these errors automatically and try to set it right at the earliest. Try refreshing the page. If the problem persists contact us at info@sunbird.org.'
           schema:
             $ref: '#/definitions/UserErrorResponse'
+  /user/v1/migrate:
+    post:
+      description:  >-
+        This API is used to migrate user from custodian channel to non custodian channel
+      summary: Migrate user
+      tags:
+        - User APIs
+      operationId: migrateUser
+      produces:
+        - application/json
+      consumes:
+        - application/json
+      parameters:
+        - name: request
+          in: body
+          required: true
+          description: >-
```

Comment:
Remove all that is written in the body.description.

Reply:
done